### PR TITLE
maths-polynomial_evalutation

### PR DESCRIPTION
**Pull request by msk4862**
Added code for evaluating a polynomail f(x) for a value of x.

### CODE REVIEW

File: maths/polynomial_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        Objective: Computes the polynomial function for a given value x. 
+                Returns that value. 
+        Input Prams: 
+            poly: tuple of numbers - value of cofficients
+            x: value for x in f(x)
+        Return: value of f(x)
+    """
+    i = 0
+    res = 0
+    for c in poly:
```
Comment (cclauss) [inline maths/polynomial_evaluation.py]: __for i, c in enumerate(poly):__  # then delete lines 14 and 10

File: maths/polynomial_evaluation.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+def evaluate_poly(poly, x):
+    """
+        Objective: Computes the polynomial function for a given value x. 
+                Returns that value. 
+        Input Prams: 
+            poly: tuple of numbers - value of cofficients
+            x: value for x in f(x)
+        Return: value of f(x)
+    """
```
Comment (cclauss) [inline maths/polynomial_evaluation.py]: Please add a doctest:
```
    >>> evaluate_poly((0.0, 0.0, 5.0, 9.3, 7.0), 10)
    79800.0
```
Then makes sure it passes locally with: __python3 -m doctest -v maths/polynomial_evaluation.py__

File: maths/polynomial_evaluation.py (hunk 1 above)
Comment (cclauss) [reply maths/polynomial_evaluation.py]: Actually, the whole function can be a one-liner: __return sum(c * x ** i for i, c in enumerate(poly))__

**Comment by msk4862** [comment]
@cclauss you can review the changes now.

**Pull request merged**